test/pthread/test_pthread_gcc_64bit_atomic_fetch_and_op.cpp

```diff
@@ -129,7 +129,7 @@ int main()
 				fetch_and_and_data = HILO(65536 + (1<<(NUM_THREADS+1))-1, (1<<(NUM_THREADS+1))-1);
 				for(int i = 0; i < NUM_THREADS; ++i)
 				{
-					threadArg[i] = DUP(~(1UL<<i));
+					threadArg[i] = DUP(~(1u<<i));
```

Comment:
Doesn't this change us from an unsigned long to plain unsigned? We seem to store this to `uint64_t`, so isn't the former better, if it makes any difference?

Reply:
Yes.  DUP() here is supposed to take an i32 and use HILO() to construct an i64 where the low and high parts are the same.

The current code which uses `UL` works for wasm32 because `UL` is i32 but on wasm64 `UL` is i64 which breaks DUP / HILO macros.

I verified this was broken on 64-bit linux too... and this was the fix there.

Reply:
Makes sense, thanks.